components.json

```diff
@@ -291,7 +291,8 @@
 		"ejs": {
 			"title": "EJS",
 			"require": ["javascript", "markup-templating"],
-			"owner": "RunDevelopment"
+			"owner": "RunDevelopment",
+			"alias": "eta"
```

Comment:
You should also give this an `aliasTitle`, similar to how `svg` is an alias of `markup` with the title `SVG`.

Reply:
Done :)